docs/modules/ROOT/pages/how-tos/deployment.adoc

```diff
@@ -4,28 +4,18 @@ The Lieutenant API is meant to be deployed on a Kubernetes cluster. It needs to
 
 The xref:lieutenant-operator::index.adoc[Lieutenant Operator] needs to be deployed on the cluster and with it the accompanying CRDs. Make sure the operator watches objects in the namespace where the API is being deployed.
 
-You can find a https://kustomize.io[Kustomize] setup for a deployment in the "deploy/" folder. A role and service account will be created for the API. The `Cluster` and `Tenant` CRs will be stored in the same namespace as the API is deployed.
+You can find a https://kustomize.io[Kustomize] setup for a deployment in the `deploy/` folder. A role and service account will be created for the API. The `Cluster` and `Tenant` CRs will be stored in the same namespace as the API is deployed.
 
 [source,shell]
 ----
 kubectl create ns $namespace
 
-kubectl -n $namespace apply -k deploy/
+kubectl -n $namespace apply -k github.com/projectsyn/lieutenant-api/deploy
 ----
 
-
+[discrete]
```

Comment:
The section heading is in order. So what is the reasoning for this? If it is to suppress the table of content, then do so in the document attributes.

Reply:
I learned that this is the way to suppress the table of content in Asciidoc, all other means I tried didn't work on our setup and I already lost quite some time with it.

Reply:
That is a pity.